_schemaV2.graphql

```diff
@@ -1496,6 +1496,30 @@ type BuyersPremium {
   percent: Float
 }
 
+type BuyersPremiumAmount {
+  # A formatted price with various currency formatting options.
+  amount(
+    decimal: String = "."
+
+    # Allows control of symbol position (%v = value, %s = symbol)
+    format: String = "%s%v"
+    precision: Int = 0
+    symbol: String
+    thousand: String = ","
+  ): String
+
+  # An amount of money expressed in cents.
+  cents: Float
```

Comment:
Just to discuss something that @yuki24 and I discussed IRL – can we make this an `Int` as cents aren't really divisible?

Reply:
I think @yuki24 was trying to explain this to me IRL (and I maybe didn't perfectly understand), but seems like this is a a byproduct of using types generated by the [`money` function](https://github.com/artsy/metaphysics/blob/4a81e2eed5ae9771352b45283e218cb6c4ce92cd/src/schema/v2/fields/money.ts#L77-L103)...I wonder if there's some context...and there is, we use float to handle very large integers that are larger than 64 bits: https://github.com/artsy/metaphysics/pull/480